modules/monitoring-modifying-retention-time-and-size-for-prometheus-metrics-data.adoc

```diff
@@ -26,6 +26,7 @@ Thus, if you set a retention size limit lower than the maximum size set for the
 If any data blocks exceed the defined retention time or the defined size limit, Prometheus purges these data blocks.
 * If you define a value for `retentionSize` and do not define `retention`, only the `retentionSize` value applies.
 * If you do not define a value for `retentionSize` and only define a value for `retention`, only the `retention` value applies.
+* If you set the `retentionSize` or `retention` value to number 0, the default settings apply. That means, retention time is set to 11 days and retention size is not set.
```

Comment:
11 days? I think that it's 15 days by default: https://github.com/openshift/cluster-monitoring-operator/blob/5342dee811a66e598aeb9a772c8000d82078a8c0/pkg/manifests/config.go#L242-L244

Reply:
@simonpasquier hmm that is weird, since 4.13 + we mention that the default is 11, only up to 4.12 we state that it is 15 days :thinking: So I suppose that is a bug in our docs?

Reply:
So yes, there is a bug in our docs: https://issues.redhat.com/browse/OBSDOCS-884 because it is assigned to Brian at the moment, I will wait and ask him once he is online if I can assign it to myself and fix this issue here, thank you!